index.js

```diff
@@ -50,6 +51,46 @@ client.on('message', message => {
         .send(MESSAGE_PREFIX + "Send '!verify' to access other channels")
         .catch(reason => console.log(reason))
     }
+    else if (text.startsWith(CONFIG.SELF_JOIN_COMMAND) || is_self_join_alias) {
+      let guild = client.guilds.get(CONFIG.GUILD_ID)
+      let msg = is_self_join_alias ? CONFIG.SELF_JOIN_ALIASES[text] : text
+      let msg_roles = msg.replace('!join', '').trim()
```

Comment:
regex match would automatically parse the role ;)

Reply:
Latest commit also parses this through regex